scripts/Deepcell_Postprocessing.ipynb

```diff
@@ -42,14 +42,17 @@
    "source": [
     "# set up file paths\n",
     "base_dir = \"../data/example_dataset/\"\n",
-    "deepcell_dir = os.path.join(base_dir, \"deepcell_output\")\n",
-    "label_dir = os.path.join(base_dir, 'label_data')\n",
+    "old_deepcell_dir = os.path.join(base_dir, \"old_deepcell_output\")\n",
+    "deepcell_dir = os.path.join(base_dir, 'deepcell_output')\n",
+    "\n",
+    "# use xarray (if you asked Noah to forward data to deepcell, set this to true)\n",
```

Comment:
We can go ahead and kill the legacy functionality. The only person in our lab who I've done this for recently in JP, and he's already made it to the extraction step, where the rest of the pipeline is the same.